nextcord/client.py

```diff
@@ -1818,6 +1818,34 @@ def add_view(self, view: View, *, message_id: Optional[int] = None) -> None:
 
         self._connection.store_view(view, message_id)
 
+    def remove_view(self, view: View, *, message_id: Optional[int] = None) -> None:
```

Comment:
`message_id` isn't used? Is it meant to be or was this a typo

Reply:
```suggestion
    def remove_view(self, view: View) -> None:
```